distribution/changelog.txt

```diff
@@ -7,6 +7,7 @@
 - Improved: [#24409] Steam installs of RCT Classic are now detected automatically.
 - Improved: [#24413] Better performance when moving the viewport on Windows and Linux platforms.
 - Improved: [#24417] Improve the fallback vehicle sprites for Dive Loops.
+- Improved: [#24467] Apply tweening only to on-screen entities at the default zoom level for better performance with uncapped FPS.
```

Comment:
If I’m not mistaken, tweening will still be done at the more zoomed _in_ levels, just not at the zoomed out ones, is that right?

Reply:
Hm right, let me think of something else.

Reply:
Adjusted it.